libfwupd/fwupd-common.c

```diff
@@ -538,7 +534,7 @@ fwupd_build_history_report_json (GPtrArray *devices, GError **error)
 	g_return_val_if_fail (error == NULL || *error == NULL, NULL);
 
 	/* get a hash that represents the machine */
-	machine_id = fwupd_build_machine_id ("fwupd", error);
```

Comment:
this change broke the fuzzing target . So fuzzing target will need to set this to something too.